code/modules/mapfluff/ruins/spaceruin_code/anomalyresearch.dm

```diff
@@ -0,0 +1,74 @@
+///Spawns a big, amped up fat anomaly
+/obj/effect/spawner/random/big_anomaly
+	name = "big anomaly spawner"
+	icon_state = "big_anomaly"
+	loot = list(
+		/obj/effect/anomaly/pyro/big,
+		/obj/effect/anomaly/flux/big,
+		/obj/effect/anomaly/bluespace/big,
+		/obj/effect/anomaly/grav/high/big
+	)
+
+///Spawns a stable anomally that doesnt drop cores and doesn't destroy or alter the environment
+/obj/effect/spawner/random/environmentally_safe_anomaly
+	name = "safe anomaly spawner"
+	icon_state = "anomaly"
+	loot = list(
+		/obj/effect/anomaly/flux,
+		/obj/effect/anomaly/bluespace,
+		/obj/effect/anomaly/hallucination,
+		/obj/effect/anomaly/bioscrambler
+	)
+
+	var/anchor_anomaly = FALSE
+
+/obj/effect/spawner/random/environmentally_safe_anomaly/make_item(spawn_loc, type_path_to_make)
+	. = ..()
+
+	var/obj/effect/anomaly/anomaly = .
+	anomaly.stabilize(anchor = anchor_anomaly, has_core = FALSE)
+
+/obj/effect/spawner/random/environmentally_safe_anomaly/immobile
+	name = "stationairy safe anomaly spawner"
+	icon_state = "anomaly_stationairy"
```

Comment:
```suggestion
	name = "stationary safe anomaly spawner"
	icon_state = "anomaly_stationary"
```

i mean it's a spawner and it shouldn't show to players but just in case
also change the dmi if you do this